RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitProducer.h

```diff
@@ -0,0 +1,39 @@
+#ifndef RecoLocalCalo_HGCalRecProducers_HGCalRecHitProducer_hh
+#define RecoLocalCalo_HGCalRecProducers_HGCalRecHitProducer_hh
+
+/** \class HGCalRecHitProducer
+ *   produce HGCAL rechits from uncalibrated rechits
+ *
+ *  simplified version of Ecal code
+ *
+ *  \author Valeri Andreev (ported to 76X by L. Gray)
+ *
+ **/
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+
+#include "RecoLocalCalo/HGCalRecProducers/interface/HGCalRecHitWorkerBaseClass.h"
+
+class HGCalRecHitProducer : public edm::stream::EDProducer<> {
+  
+ public:
+  explicit HGCalRecHitProducer(const edm::ParameterSet& ps);
+  ~HGCalRecHitProducer();
+  virtual void produce(edm::Event& evt, const edm::EventSetup& es);
```

Comment:
add override here

... the whole .h file is not really needed and can be merged into .cc